conformance-test/v4SignedUrl.ts

```diff
@@ -114,7 +114,7 @@ describe('v4 conformance test', () => {
         const fakeTimer = sinon.useFakeTimers(NOW);
         const bucket = storage.bucket(testCase.bucket);
         const expires = NOW.valueOf() + testCase.expiration * 1000;
-        const version = 'v4' as 'v4';
+        const version = 'v4' as const;
```

Comment:
@bcoe Why did this change?

Reply:
I believe that this is our linter deciding that this is how it would like to represent constants that only have one possible value.